recipe/meta.yaml

```diff
@@ -41,11 +41,18 @@ requirements:
     # - libscotch # no 64bit support
     - libhwloc
     - metis
+    - python
```

Comment:
do you need python to generate files? if so put it in the build section.

Otherwise, if you need for some runtime stuff, you might need cross-python as well.

Copy over from: https://github.com/conda-forge/scipy-feedstock/blob/9a03cf28abe8bd9fd71f63d873891771d4c7fa32/recipe/meta.yaml#L51

Reply:
OSX Arm is cross compiled and has no emulation support, so we need be more careful.

Reply:
it's mainly copieng and rendering of files. See here for example:
https://github.com/looooo/pastix/blob/master/wrappers/python/CMakeLists.txt#L35L37

Reply:
So you should add it to the build section
https://github.com/looooo/pastix/blob/master/wrappers/python/CMakeLists.txt#L16

and just "pin it" so some reasonably modern version, like 3.11 so as to avoid the whole build matrix.

Reply:
It seems that it does install things in the site packages dir, so you need the cross python stuff.

```
2024-08-25T13:03:03.5925887Z ?rw-rw-r-- 0/0       1022 2024-07-12 14:23:22 lib/python3.12/site-packages/pastix/__init__.py 
2024-08-25T13:03:03.5926357Z ?rw-rw-r-- 0/0      19318 2024-07-12 14:23:22 lib/python3.12/site-packages/pastix/__pastix__.py 
2024-08-25T13:03:03.5926912Z ?rw-rw-r-- 0/0       1218 2024-08-25 13:02:37 lib/python3.12/site-packages/pastix/__pycache__/__init__.cpython-312.pyc 
2024-08-25T13:03:03.5927469Z ?rw-rw-r-- 0/0      25708 2024-08-25 13:02:37 lib/python3.12/site-packages/pastix/__pycache__/__pastix__.cpython-312.pyc 
2024-08-25T13:03:03.5928030Z ?rw-rw-r-- 0/0      10619 2024-08-25 13:02:37 lib/python3.12/site-packages/pastix/__pycache__/enum.cpython-312.pyc 
2024-08-25T13:03:03.5928588Z ?rw-rw-r-- 0/0       7637 2024-08-25 13:02:37 lib/python3.12/site-packages/pastix/__pycache__/pastix.cpython-312.pyc 
2024-08-25T13:03:03.5929128Z ?rw-rw-r-- 0/0      10662 2024-08-25 13:02:37 lib/python3.12/site-packages/pastix/__pycache__/solver.cpython-312.pyc 
2024-08-25T13:03:03.5929625Z ?rw-rw-r-- 0/0       7189 2024-08-25 13:01:23 lib/python3.12/site-packages/pastix/enum.py 
2024-08-25T13:03:03.5930204Z ?rw-rw-r-- 0/0       4413 2024-07-12 14:23:22 lib/python3.12/site-packages/pastix/pastix.py 
2024-08-25T13:03:03.5930662Z ?rw-rw-r-- 0/0       6733 2024-07-12 14:23:22 lib/python3.12/site-packages/pastix/solver.py 
2024-08-25T13:03:03.5931141Z ?rw-rw-r-- 0/0       1229 2024-07-12 14:23:24 lib/python3.12/site-packages/spm/__init__.py 
2024-08-25T13:03:03.5931642Z ?rw-rw-r-- 0/0       1049 2024-08-25 13:02:37 lib/python3.12/site-packages/spm/__pycache__/__init__.cpython-312.pyc 
2024-08-25T13:03:03.5932282Z ?rw-rw-r-- 0/0      22977 2024-08-25 13:02:37 lib/python3.12/site-packages/spm/__pycache__/__spm__.cpython-312.pyc 
2024-08-25T13:03:03.5932834Z ?rw-rw-r-- 0/0       6387 2024-08-25 13:02:37 lib/python3.12/site-packages/spm/__pycache__/enum.cpython-312.pyc 
2024-08-25T13:03:03.5933354Z ?rw-rw-r-- 0/0      15304 2024-08-25 13:02:37 lib/python3.12/site-packages/spm/__pycache__/spm.cpython-312.pyc 
2024-08-25T13:03:03.5933860Z ?rw-rw-r-- 0/0      15157 2024-07-12 14:23:24 lib/python3.12/site-packages/spm/__spm__.py 
2024-08-25T13:03:03.5934310Z ?rw-rw-r-- 0/0       3266 2024-08-25 13:01:20 lib/python3.12/site-packages/spm/enum.py 
2024-08-25T13:03:03.5934778Z ?rw-rw-r-- 0/0      10660 2024-07-12 14:23:24 lib/python3.12/site-packages/spm/spm.py
```